### PR TITLE
Revert "fix(syslog-ng): add support for multi-line logs"

### PR DESCRIPTION
**Pull request by fruch**
Seem like when running older versions of scylla 4.6 and lower
which we still use as oracles for gemini tests.

we fail to install syslog-ng, and fallback to using rsyslog
the way we configure rsyslog isn't comptible with syslog-ng
rsyslog driver.

we need to maybe put it behind a configuration option,
or invest in removing the rsyslog support completly
and fixing the install issues syslog-ng had, across the board

This reverts commit a11dbb24f9e9754872b1c47138aed5fe4badae85.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by roydahan** [comment]
Why can't we install syslog-ng with older versions?
What's the issue there?

**Comment by fruch** [comment]
> Why can't we install syslog-ng with older versions? What's the issue there?

I'm talking about images using centos8, before we moved to ubuntu.

it's race condition between our boot script and cloud-init, to when the image repos are setup correctly to AWS region mirrors.
scylla-machine-image is blocking the cloud-init from finishing.

we have this issue also with ubutnu, but it happens very rarely

meanwhile on newer images we switch to doing this configuration as cloud-init script and it working better, but we can't do that with older versions since the support for user-data including cloud-init and scylla own configuration is only on master, and would be available on 5.1 and forward.


long story short, if we won't revert I see that gemini jobs would fail since the oracles are old.if you are o.k. with that and rather have someone change that issue on the older images, or switch oracles to 4.6/5.0 to see if it would be better ?

**Comment by fruch** [comment]
@roydahan 

I've might have found a solution to align also the rsyslog to that change, so we shouldn't need to revert:
https://github.com/fruch/scylla-cluster-tests/commit/91a74cc105aa2cd2c57520ce3db07f4c9e595219

I'm trying it out in this job:
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/gemini-3h-test/7/

fixing the way we run the boot script is in motion, but it would take a while until we can remove some of the hack and fallback that are in place, https://github.com/scylladb/scylla-cluster-tests/pull/5035 was a start going in that direction.

**Comment by fruch** [comment]
> @roydahan
> 
> I've might have found a solution to align also the rsyslog to that change, so we shouldn't need to revert: [fruch@91a74cc](https://github.com/fruch/scylla-cluster-tests/commit/91a74cc105aa2cd2c57520ce3db07f4c9e595219)
> 
> I'm trying it out in this job: https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/gemini-3h-test/7/
> 
> fixing the way we run the boot script is in motion, but it would take a while until we can remove some of the hack and fallback that are in place, #5035 was a start going in that direction.

my fix didn't worked (would need to invest more time into it making it work)

@roydahan I'm reverting, and we'll later try to handle it offline.  (not on SCT master)
